src/css/main.css

```diff
@@ -736,6 +743,10 @@ a.help-link:hover {
 
 /* This is a temporaly hack, we should style the editor instead of overwriting
    the a-scene to fix the "display: block" issue. */
-a-scene {
+.aframe-inspector-opened a-scene {
```

Comment:
What about multiple scenes? Maybe we should keep track of the actively inspected scene.